### PR TITLE
Update Bangladesh README

### PR DESCRIPTION
**Pull request by turnerm**
Since we're cleaning up the READMEs, updated the Bangladesh one to better reflect the present situation

### CODE REVIEW

File: analyses/bgd/README.md [hunk 1]
```diff
@@ -1,33 +1,81 @@
 # Bangladesh Anticipatory Action
 
-## Background information
+[![Generic badge](https://img.shields.io/badge/STATUS-ENDORSED-%231EBFB3)](https://shields.io/)
```
Comment (hannahker) [inline analyses/bgd/README.md:3]: Could you add a 'Last updated' line to go along with this?

File: analyses/bgd/README.md [hunk 2]
```diff
@@ -40,33 +88,65 @@ The content within this repository is structured as follows:
 │   └── config.yml             <- Parameters to configure the analysis
 │
 ├── trigger_development       <- Scripts related to the trigger analysis
-│
+│   ├── utils             
+│   │   └── utils.py          <- Utiltiy methods for the trigger development scripts and notebooks 
+│   ├── 01_get_glofas_data.py  <- [defunct] Download GloFAS data
```
Comment (hannahker) [inline analyses/bgd/README.md:93]: I'd argue that this level of detail here probably isn't necessary and might be tough to maintain. Could probably be fine letting each of the notebook titles speak for themselves?

**Review by hannahker — COMMENTED** [review]
Just a couple relatively minor comments but looks great otherwise. Thanks for updating!

File: analyses/bgd/README.md (hunk 1 above)
Comment (turnerm) [reply analyses/bgd/README.md:3]: Oops yes, done! And I also mentioned the activation

File: analyses/bgd/README.md (hunk 2 above)
Comment (turnerm) [reply analyses/bgd/README.md:93]: Yeah good point, I'll take these out!

**Pull request merged**